examples/cc3220/xively_demo/xively_example.c

```diff
@@ -341,7 +346,7 @@ void * xivelyExampleThread( void *arg )
 	}
 
 	/* start simplelink here so we can use the simplelink file API */
-	sl_Start(NULL, NULL, NULL);
+	retval = sl_Start(0, 0, 0);
```

Comment:
You're right. Most examples in the SDK use `0`s, but these are pointers after all...